libs/ledger-live-common/src/families/celo/api/hubble.ts

```diff
@@ -83,10 +83,17 @@ const transactionToOperation = (
   const fee = new BigNumber(transaction.data?.gas_used || 0).times(
     new BigNumber(transaction.data?.gas_price || 0)
   );
-  const value =
-    type === "LOCK"
-      ? new BigNumber(transaction.amount).plus(fee)
-      : new BigNumber(transaction.amount);
+
```

Comment:
Like other assets (e.g. Cosmos), we need to set the value of staking operations to the fee.  Otherwise, it messes up the account balance chart.